data/tests_jobfactory.py

```diff
@@ -71,3 +72,22 @@ def test_favors_user_inputs(self):
         job = job_factory.create_job()
         expected_job_order = json.dumps({'input1':'user'})
         self.assertEqual(expected_job_order, job.job_order)
+
+    def test_calculate_stage_group_size(self):
+        stage_group = JobFileStageGroup.objects.create(user=self.user)
+        dds_file_sizes = [10 * 1024 * 1024,  # 10 MB
+                          2.5 * 1024 * 1024 * 1024,  # 2.5 GB
+                          ]
+        url_file_sizes = [0.5 * 1024 * 1024 * 1024]  # 0.5 GB
+        for size in dds_file_sizes:
+            DDSJobInputFile.objects.create(stage_group=stage_group, dds_user_credentials=self.worker_cred, size=size)
+        for size in url_file_sizes:
+            URLJobInputFile.objects.create(stage_group=stage_group, size=size)
+        self.assertEqual(3, int(calculate_stage_group_size(stage_group)))
```

Comment:
Can this be more precise? This would pass even if the 10MB wasn't considered. Maybe increase that file size or avoid truncating to `int`?